Algorithm.CSharp/BasicTemplateAlgorithm.cs

```diff
@@ -26,22 +26,22 @@ namespace QuantConnect.Algorithm.CSharp
     /// <meta name="tag" content="trading and orders" />
     public class BasicTemplateAlgorithm : QCAlgorithm
     {
-        private Symbol _spy = QuantConnect.Symbol.Create("SPY", SecurityType.Equity, Market.USA);
+        private Symbol _bhp = QuantConnect.Symbol.Create("BHP", SecurityType.Equity, Market.AU);
 
         /// <summary>
         /// Initialise the data and resolution required, as well as the cash and start-end dates for your algorithm. All algorithms must initialized.
         /// </summary>
         public override void Initialize()
         {
-            SetStartDate(2013, 10, 07);  //Set Start Date
+            SetStartDate(2011, 10, 07);  //Set Start Date
             SetEndDate(2013, 10, 11);    //Set End Date
             SetCash(100000);             //Set Strategy Cash
 
             // Find more symbols here: http://quantconnect.com/data
             // Forex, CFD, Equities Resolutions: Tick, Second, Minute, Hour, Daily.
             // Futures Resolution: Tick, Second, Minute
             // Options Resolution: Minute Only.
-            AddEquity("SPY", Resolution.Minute);
+            AddEquity("BHP", Resolution.Daily);
```

Comment:
These changes appear to be for local testing. Further, these will break the regression tests. Please remove these types of changes from the PR.

Reply:
If keeping this is critical, we could create a different `BasicAustralianMarketTemplateAlgorithm` or something similar.

Reply:
I do think keeping this is important as we are attempting to get Lean working on foreign equities and as such we need an algo attempting to run on these foreign equities as we try and configure everything. That said - it was silly of me to overwrite BasicTemplateAlgorithm like this instead of creating a new algo like you suggested. I will add a BasicAustralianMarketTemplateAlgorithm and revert BasicTemplateAlgorithm.

In regards to the meow.txt - that was just me being silly haha :)